torch/nested/_internal/ops.py

```diff
@@ -1138,17 +1143,27 @@ def native_layer_norm_default(func, *args, **kwargs):
 
     if (
         -1 in normalized_shape
-    ):  # special handling for normalizing over the ragged dimension
+    ):  # special handling for normalizing over the ragged dimension, assuming that -1 represents the ragged shape
```

Comment:
I don't think this handling is correct:

Say we have a (A, B, j0, C, D) tensor and we want to take the layernorm with normalized_shape (j0, C, D); this will actually take normalized_shape (B, j0, C, D).